scipy/stats/stats.py

```diff
@@ -3339,9 +3339,45 @@ def pointbiserialr(x, y):
     pvalue : float
         2-tailed p-value
 
+    Notes
+    -----
+    For testing the hypothesis that the point-biserial correlation coefficient
+    is significantly different from zero, a t-test with n-2 degrees of freedom
+    should be used. Pearsonr is equivalent.
```

Comment:
This looks like a suggestion to users to use a ``ttest_*`` function, while it tries to explain the algorithm, So I'd rephrase this as:

    `pointbiserialr` uses a t-test with ``n-2`` degrees of freedom.
    It is equivalent to `pearsonr`.